journald/journald.hpp

```diff
@@ -32,6 +37,19 @@ struct Flags : public virtual flags::FlagsBase
       "before it is written to journald.  See '--labels'."
       "\n");
 
+    add(&Flags::destination_type,
+        "destination_type",
+        "Determines where logs should be piped.\n"
+        "Valid destinations include: 'journald', 'sandbox', or 'both'.",
+        "journald",
```

Comment:
Why is the module default "journald", while the proposed dcos default is "both"? Shouldn't these be consistent, or just set/defaulted in a single place?